src/code/z_viscvg.c

```diff
@@ -75,20 +109,29 @@ void func_800ACE98(struct_801664F0* this, Gfx** gfxp) {
     }
 
     switch (this->type) {
-        case 1:
-            gSPDisplayList(gfx++, D_8012AC40);
+        case FB_FILTER_CVG_RGB:
+            gSPDisplayList(gfx++, sCoverageRGBDL);
             break;
-        case 2:
-            gDPSetColor(gfx++, G_SETPRIMCOLOR, this->color.rgba);
-            gSPDisplayList(gfx++, D_8012AC58);
+
+        case FB_FILTER_CVG_RGB_UNIFORM:
```

Comment:
Looks like this needs to be renamed to `FB_FILTER_CVG_RGB_PRIMFOG` (it causes the build to fail currently)